.github/dependabot.yml

```
@@ -1,38 +1,25 @@
-
 version: 2
 updates:
-
-  - package-ecosystem: "gitsubmodule"
-    directory: "/"
-    schedule:
-      interval: "daily"
-    target-branch: "develop"
-
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: "daily"
     target-branch: "develop"
 
-  - package-ecosystem: "docker"
-    directory: "/build"
-    schedule:
-      interval: "daily"
-    target-branch: "develop"
-
   - package-ecosystem: "gomod"
     directory: "/build"
     schedule:
       interval: "daily"
     target-branch: "develop"
 
-  - package-ecosystem: "maven"
-    directory: "/build"
+  - package-ecosystem: "gomod"
+    directory: "/website"
     schedule:
       interval: "daily"
-    target-branch: "develop"
+    target-branch: "main"
 
   - package-ecosystem: "npm"
+    directory: /
     schedule:
       interval: "daily"
     target-branch: "develop"
```

This is also used by the website, but needs to be at the top-level for Hugo to find it.

```suggestion
    target-branch: "main"
```

---

On second thought, I don't believe we should be making any changes to main directly. They should flow through `develop`.